package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@govtechsg/oa-encryption": "^1.3.5",
     "@tradetrust-tt/tt-verify": "^8.2.0",
```

Comment:
It would be good to update "@tradetrust-tt/tt-verify" first before updating this repo, to have the consistency of having the latest package in the repo.

Reply:
added